yarn.lock

```diff
@@ -0,0 +1,8 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
```

Comment:
similarly, this should not be leaked to the root directory. maybe it's the way we are running the build scripts?